app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionContentViewModel.kt

```diff
@@ -9,7 +9,8 @@ class SelectionInteractionContentViewModel(
   val htmlContent: SubtitledHtml,
   val hasConversationView: Boolean,
   private val itemIndex: Int,
-  private val selectionInteractionViewModel: SelectionInteractionViewModel
+  private val selectionInteractionViewModel: SelectionInteractionViewModel,
+  val enabledItem: ObservableBoolean
```

Comment:
Perhaps instead: ``isEnabled`` would be clearer?

Reply:
Okay ben